### PR TITLE
Make webnn-polyfill.js importable

### PR DESCRIPTION
**Pull request by Honry**
And prepare for npm publish.

### CODE REVIEW

**Comment by Honry** [comment]
@BruceDai, PTAL, thanks!

**Comment by BruceDai** [comment]
LGTM, with a nit, thanks.

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "description": "WebNN API polyfill",
   "main": "dist/webnn-polyfill.js",
+  "jsdelivr": "dist/webnn-polyfill.js",
+  "unpkg": "dist/webnn-polyfill.js",
```
[the remark this answers is not in the document]
Comment (Honry) [reply package.json:7]: This is a necessary comma.

[the remark this answers is not in the document]
Comment (BruceDai) [reply package.json:7]: Right, I missed it.

**Pull request merged**